src/collectors/mesos/test/testmesos.py

```diff
@@ -164,6 +165,12 @@ def test_https(self):
         self.assertEqual('https://localhost:5050/metrics/snapshot',
                          self.collector._get_url("metrics/snapshot"))
 
+    def test_sum_statistics(self):
+        results = json.load(self.getFixture('slave_monitor_statistics.json'))
+        sum = {}
+        for i in results:
+            sum = self.collector._sum_statistics(sum,i['statistics'])
```

Comment:
can you add `self.assertEqual(XXX, sum)`? (XXX being the sum that should be there)